test/browser_tests/spec_suites/shared/shared_events.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var events = require( '../../page_objects/page_events.js' );
+var EventsPage = events.EventsPage;
+var ArchivePage = events.ArchivePage;
+
+describe( 'Events Landing page', function() {
+  var googleAPI = 'https://maps.googleapis.com/maps/api/staticmap?';
+  var page;
+
+  beforeEach( function() {
+    page = new EventsPage();
+    page.get();
+  } );
+
+  it( 'should properly load in a browser', function() {
+    expect( page.pageTitle() ).toBe( 'Upcoming Events' );
+  } );
+
+  it( 'should include a hero', function() {
+    expect( page.heroElem.isPresent() ).toBeTruthy();
+    expect( page.hero.map.getAttribute( 'style' ) ).toContain( googleAPI );
+    expect( page.hero.heading.getText() ).toBeDefined();
+    expect( page.hero.date.getText() ).toBeDefined();
+    expect( page.hero.time.getText() ).toBeDefined();
+  } );
+
+  it( 'should include at least one event', function() {
+    expect( page.events.count() ).toBeGreaterThan( 0 );
+  } );
+
+  it( 'should include all event meta data', function() {
```

Comment:
I disagree with this currently because we reload the page on each expectation, which makes the tests take even longer. We should consider not loading the page when tests are static in the `beforeEach()` but that sounds like a discussion outside this PR.